src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/library_view.cljs

```diff
@@ -15,18 +15,20 @@
         (common/attribute-list? value) (clojure.string/join ", " (common/attribute-values value))
         :else value))
 
-(defn- render-library-row [key value]
+(defn- render-library-row [key value doid]
   (when value
     [:div {:style {:display "flex" :padding "0.5em 0" :borderBottom (str "2px solid " (:line-default style/colors))}}
      [:div {:style {:flexBasis "33%" :fontWeight "bold" :paddingRight "2em"}} key]
-     [:div {:style {:flexBasis "67%"}} value]]))
+     [:div {:style {:flexBasis "67%"}} value
+      (if (not (nil? doid)) [:span {:style {:fontStyle "italic"}} (str " (" doid ")")])]]))
 
 
 (defn- render-property [library-schema library-attributes property-key]
   (render-library-row
    (get-in library-schema [:properties property-key :title])
-   (render-value 
-    (or (get library-attributes property-key) (get-in library-schema [:properties property-key :default])))))
+   (render-value
+    (or (get library-attributes property-key) (get-in library-schema [:properties property-key :default])))
+    (if (= (get-in library-schema [:properties property-key :typeahead]) "ontology") (get library-attributes :library:diseaseOntologyID) nil)))
```

Comment:
1) un-indent this line - it should be parallel to the `(render-value` and `(get-in` preceding it
2) this still has a hardcoded `:library:diseaseOntologyID`. It should be ... guessing here ... `(get-in library-schema [:properties property-key : relatedID])`